pkg/shp/cmd/build/delete.go

```diff
@@ -50,12 +58,35 @@ func (c *DeleteCommand) Validate() error {
 }
 
 // Run contains main logic of delete subcommand
-func (c *DeleteCommand) Run(params *params.Params) error {
+func (c *DeleteCommand) Run(params *params.Params, io *genericclioptions.IOStreams) error {
 	var b buildv1alpha1.Build
 
-	if err := buildResource.Get(c.name, &b); err != nil {
+	br := resource.GetBuildResource(params)
+	brr := resource.GetBuildRunResource(params)
+
+	if err := br.Get(c.cmd.Context(), c.name, &b); err != nil {
+		return err
+	}
+
+	if err := br.Delete(c.cmd.Context(), c.name); err != nil {
 		return err
 	}
 
-	return buildResource.Delete(c.name)
+	if c.deleteRuns {
+		label := fmt.Sprintf("build.build.dev/name=%v", c.name)
```

Comment:
It is probably safe to just replace the `LabelSelector: label,` with `LabelSelector: fmt.Sprintf(...),` instead of creating this separate variable.

Reply:
TODO: I also had it changed to `const` from `build`, but it probably got lost during the extensive cherry pick after the previous PR